coral/src/app/layout/main-navigation/MainNavigation.test.tsx

```diff
@@ -292,4 +222,37 @@ describe("MainNavigation.tsx", () => {
       });
     });
   });
+
+  describe("user can see their current team", () => {
+    beforeEach(() => {
+      customRender(<MainNavigation />, {
+        memoryRouter: true,
+        queryClient: true,
+      });
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      //@ts-ignore
+      useQuerySpy.mockReturnValue({ data: "Team-name", isLoading: true });
+    });
```

Comment:
You'll need to do the mocking BEFORE the rendering, so move the mocking above the rendering